sdk/src/main/java/com/hcaptcha/sdk/HCaptcha.java

```diff
@@ -37,7 +38,11 @@ private HCaptcha(@NonNull final Context context) {
      * @return new {@link HCaptcha} object
      */
     public static HCaptcha getClient(@NonNull final Context context) {
-        return new HCaptcha(context);
+        return new HCaptcha(context, HCaptchaInternalConfig.builder().build());
+    }
+
+    static HCaptcha getClient(@NonNull final Context context, HCaptchaInternalConfig internalConfig) {
```

Comment:
I changed to favor passing the `internalConfig` via constructor instead of duplicating public methods `setup` and `verifyWithHCaptcha`. @CAMOBAP